.script/contentValidator.ts

```diff
@@ -9,11 +9,22 @@ export async function ValidateFileContent(filePath: string): Promise<ExitCode>
     {
         const fileContent = fs.readFileSync(filePath, "utf8");
         const searchText = "Azure Sentinel"
-        const hasAzureSentinelText = fileContent.toLowerCase().includes(searchText.toLowerCase());
-        
+        const replaceText = '"targetProduct": "Azure Sentinel"';
+
+        const hasTargetProductAzureSentinel = fileContent.includes(replaceText);
+        const replacedFileContent = fileContent.replace(replaceText, "");
+        const hasAzureSentinelText = replacedFileContent.toLowerCase().includes(searchText.toLowerCase());
+
         if (hasAzureSentinelText)
         {
-            throw new Error(`Please update text from 'Azure Sentinel' to 'Microsoft Sentinel' in file '${filePath}'`);
+            if (hasTargetProductAzureSentinel)
+            {
+                throw new Error(`Please update text from 'Azure Sentinel' to 'Microsoft Sentinel' except 'targetProduct' key-value pair in file '${filePath}'`);
```

Comment:
Should this line really throw an error? I would think it should only emit log message i.e. console.log(`Please update text from 'Azure Sentinel' to 'Microsoft Sentinel' except 'targetProduct' key-value pair in file '${filePath}'`)